### PR TITLE
feat: add Webrecorder profile

### PR DESCRIPTION
**Pull request by edsu**
I don't know if it is common practice for respec users to push profiles back upstream. But if it is here is a profile that the [Webrecorder](https://webrecorder.net) project is [using](https://webrecorder.github.io/wacz-spec/1.1.1/).

### CODE REVIEW

**Comment by edsu** [comment]
Feedback on this PR would be appreciated, even if the answer is to close it because sharing profiles is not something that is desirable.

File: src/webrecorder/abstract.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
+// @ts-check
+// Module w3c/abstract
```
Comment (marcoscaceres) [inline src/webrecorder/abstract.js:2]:
```suggestion
// Module webrecorder/abstract
```

File: src/webrecorder/conformance.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
+// @ts-check
+// Module w3c/conformance
```
Comment (marcoscaceres) [inline src/webrecorder/conformance.js:2]:
```suggestion
// Module webrecorder/conformance
```

**Comment by marcoscaceres** [comment]
Hi @edsu, including it should be fine. Would you mind removing the changes to the `build/` directory? Those should only get generated during deployment.

File: profiles/webrecorder.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  import("../src/core/dfn.js"),
+  import("../src/core/pluralize.js"),
+  import("../src/core/examples.js"),
+  import("../src/core/issues-notes.js"),
+  import("../src/core/best-practices.js"),
+  import("../src/core/figures.js"),
+  import("../src/core/webidl.js"),
+  import("../src/core/biblio.js"),
+  import("../src/core/link-to-dfn.js"),
+  import("../src/core/xref.js"),
```
Comment (sidvishnoi) [inline profiles/webrecorder.js:30]: You might not need `xref`, `ui/search-xref` modules, unless you reference definitions and terms from W3C specs. This should reduce bundle size and avoid unintended dependency on W3C related stuff.

Also potentially:
- `mdn-annotation`
- `webidl`
- `webidl-index`
- `linter-rules/wpt-tests-exist`.

**Comment by sidvishnoi** [comment]
Also, please add `webrecorder` to the release script so it gets added to `./builds/respec-webrecorder.js` during release:
https://github.com/w3c/respec/blob/740123075705afc1eae4093eac06c9b33255a2d1/tools/release.js#L363

And also to: https://github.com/w3c/respec/blob/740123075705afc1eae4093eac06c9b33255a2d1/tools/netlify.js#L13

**Comment by marcoscaceres** [comment]
Closing due to inactivity. Please feel free to send again though. We sent some suggestions above.

**Pull request closed without merging**